### PR TITLE
added feedback for delete

### PR DESCRIPTION
**Pull request by myung03**
Added feedback for when user deletes a location

### CODE REVIEW

File: interface/locales/en/common.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "Location added successfully": "Location added successfully.",
```
Comment (utkubakir) [inline interface/locales/en/common.json:2]: missed this

File: interface/locales/en/common.json [hunk 2]
```diff
@@ -388,7 +389,9 @@
   "local_locations": "Local Locations",
   "local_node": "Local Node",
   "location": "Location",
+  "location_added_successfully": "Location added successfully. ",
```
Comment (utkubakir) [inline interface/locales/en/common.json:392]: extra space

**Review by utkubakir — CHANGES_REQUESTED** [review]

File: interface/locales/en/common.json (hunk 1 above)
Comment (myung03) [reply interface/locales/en/common.json:2]: having issues with adding the toast to location actually - trying to add the toast.success in the try part of the try catch but not working? any idea why

Comment (utkubakir) [reply interface/locales/en/common.json:2]: not sure where. happy to move this discussion to slack with more information on where in the code

**Review by utkubakir — APPROVED** [review]

**Pull request merged**